okio/src/commonMain/kotlin/okio/internal/ByteString.kt

```diff
@@ -288,6 +289,10 @@ internal fun ByteString.commonWrite(buffer: Buffer, offset: Int, byteCount: Int)
   buffer.write(data, offset, byteCount)
 }
 
+internal fun ByteString.commonDigest(algorithm: String) = ByteString(
+  newMessageDigest(algorithm).apply { update(data) }.digest()
```

Comment:
FWIW, this is too much stuff in one line for me

Reply:
I'll break it up